Ginger/GingerCore/Resources/GingerHTMLRecorder.js

```diff
@@ -46,7 +46,7 @@ function define_GingerRecorderLib() {
 
             if (!isActionFromTimeOut) {
                 element = event.currentTarget;
-                GingerRecorderLib.AddRecordingtoQ(element, "Click", element.value, element.type);
+                GingerRecorderLib.AddRecordingtoQ(element, "Submit", element.value, "Button");
 
```

Comment:
Need to do same as done here, in which element.type was replaced with eElementType enum "Button" to all rest below, so JS will return the actual  eElementType enum needed and no extra mapping functions will be needed in SelenuimDriver side